contrib/test.sh

```diff
@@ -62,6 +62,8 @@ do
     cargo test --verbose --features="$feature"
 done
 
+cargo run --example ecdsa-psbt --features=bitcoinconsensus
```

Comment:
Not related to this issue, but I just noted that other examples are not being when "DO_NO_STD" is not set